src/client/logging/transports.ts

```diff
@@ -71,6 +73,33 @@ export function getConsoleTransport(formatter: logform.Format): Transport {
     });
 }
 
+export function setPythonOutputChannelTransport(outputChannel: IOutputChannel) {
+    pythonOutputChannel = outputChannel;
+}
+
+class PythonOutputChannelTransport extends Transport {
+    // tslint:disable-next-line: no-any
+    constructor(options?: any) {
+        super(options);
+    }
+    // tslint:disable-next-line: no-any
+    public log?(info: { level: string; message: string; [formattedMessage]: string }, next: () => void): any {
```

Comment:
We don't use "level" here at all, right?
```suggestion
    public log?(info: { message: string; [formattedMessage]: string }, next: () => void): any {
```